components/2016/legacyProfileRouter.ts

```diff
@@ -155,7 +147,6 @@ legacyProfileRouter.post(
                 MustBeSaved: progression.MustBeSaved,
             })
         }
-        // TODO: HELP! Please DM rdil if you see this
```

Comment:
(Vague todo I wrote for myself when realizing how big of a pain pro1 challenges would be)